app/reducers/currentpatient.js

```diff
@@ -2,20 +2,20 @@ import axios from 'axios';
 
```

Comment:
per my comment at top, did you intentionally modify currentpatient.js or is this just an artifact?